lib/jira.js

```diff
@@ -47,10 +47,10 @@ module.exports = class {
   }
 
   async getUsersFromAccountIds(accountIds) {
-    const query = { accountId: accountIds };
+    const query = { maxResults: 25, accountId: accountIds };
```

Comment:
Why is maxResults needed?

Reply:
@himichaelroberts it defaulted to 10, so I wanted to have a safer, higher number